### PR TITLE
[cp-zookeeper] - Add Custom env parameter at helm charts

### PR DESCRIPTION
**Pull request by cesarokuti**
## What changes were proposed in this pull request?

Propose to add customEnvs in cp-zookeeper helm/charts. For example change log level and others envs.

## How was this patch tested?

This patch are tested in Kubernetes EKS env.

### CODE REVIEW

**Comment by cesarokuti** [comment]
[clabot:check]

**Comment by vinixavier** [comment]
+1

File: Chart.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Confluent Platform Community Edition
 name: cp-helm-charts
-version: 0.4.1
+version: 0.4.2
```
Comment (gAmUssA) [inline Chart.yaml:5]: kindly don't update the version here.
I'm doing version change with confluent platform release (which is currently 5.4.1).
Thanks

Comment (cesarokuti) [reply Chart.yaml]: Remove Version config.

File: charts/cp-zookeeper/values.yaml [hunk 2]
```diff
@@ -48,6 +48,11 @@ autoPurgePurgeInterval: 24
 ## Zookeeper JVM Heap Option
 heapOptions: "-Xms512M -Xmx512M"
 
+## Additional env variables
+customEnv: {}
```
Comment (gAmUssA) [inline charts/cp-zookeeper/values.yaml:52]: @cesarokuti hi
other charts use configOverrides convention. 
kindly follow it here. 

https://github.com/confluentinc/cp-helm-charts/blob/bc75a82288e3b9fe9970b8886e5914e5bb0e68e0/charts/cp-ksql-server/templates/deployment.yaml#L93

Comment (cesarokuti) [reply charts/cp-zookeeper/values.yaml:52]: The other charts has 2 options like:
https://github.com/confluentinc/cp-helm-charts/blob/master/charts/cp-schema-registry/templates/deployment.yaml#L91

Comment (cesarokuti) [reply charts/cp-zookeeper/values.yaml:52]: Maybe are two diferent options like Schema Registry or Kafka configurations:

https://github.com/confluentinc/cp-helm-charts/blob/bc75a82288e3b9fe9970b8886e5914e5bb0e68e0/charts/cp-kafka/templates/statefulset.yaml#L128

Comment (cesarokuti) [reply charts/cp-zookeeper/values.yaml:58]: These commit add two options like cp-kafka chart.

**Pull request closed without merging**